nanoFramework.Device.Bluetooth/GenericAttributeProfile/GattLocalCharacteristic.cs

```diff
@@ -57,16 +59,17 @@ public sealed class GattLocalCharacteristic
         /// <param name="args"></param>
         public delegate void GattLocalCharacteristicClientsChangedEventHandler(GattLocalCharacteristic sender, object args);
 
-        internal GattLocalCharacteristic(Guid characteristicUuid, GattLocalCharacteristicParameters parameters)
+        internal GattLocalCharacteristic(Guid characteristicUuid, GattLocalCharacteristicParameters parameters, INativeDevice nativeDevice)
         {
+            _nativeDevice = nativeDevice;
             _characteristicUuid = characteristicUuid.ToByteArray();
             _writeProtectionLevel = parameters.WriteProtectionLevel;
             _readProtectionLevel = parameters.ReadProtectionLevel;
 
             _properties = parameters.CharacteristicProperties;
 
             _staticValue = parameters.StaticValue;
-
+            
```

Comment:
please use 'ctrl + k + d' in Visual Studio so it will properly format the lines and remove extra spaces